src/Sarif/Schemata/sarif-2.1.0-rtm.6.json

```diff
@@ -1709,24 +1709,28 @@
         "startLine": {
           "description": "The line number of the first character in the region.",
           "type": "integer",
+          "default": 0,
```

Comment:
>default [](http://example.com/codeflow?start=11&length=7)

something is wrong here. if this is the default, then minimum should be set as 0 as well.